pages/en/lb3/Using-database-transactions.md

```diff
@@ -29,7 +29,80 @@ A LoopBack model can perform operations in a transaction when the model is atta
 
 ## Transaction APIs
 
-See the [API reference](http://apidocs.loopback.io/loopback-datasource-juggler/#transactionmixin) for full transaction API documentation.
+As of [v3.12.0](https://github.com/strongloop/loopback-datasource-juggler/tree/v3.12.0) of DataSource Juggler, there are two ways to handle transactions in LoopBack:
+
+* The [`TransactionMixin`](http://apidocs.loopback.io/loopback-datasource-juggler/#transactionmixin) gives access to the lower-level transaction API, leaving it up to the user to create and manage transaction objects, commit them on success or roll them back at the end of all intended operations.  
+  See [Lower-level Transaction API](#lower-level-transaction-api) below for more details.
+
+* To simplify and streamline the handling of transactions, [`dataSource.transaction()`](https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource-prototype-transaction) offers a higher-level abstraction over the lower-level transaction API through a single method.  
+  See [Higher-level Transaction API](#higher-level-transaction-api) below for more details.
+
+## Higher-level Transaction API
+
+Inspired by the way [Objection.js](https://github.com/Vincit/objection.js) is handling transactions by [binding them to models](http://vincit.github.io/objection.js/#binding-models-to-a-transaction), the method [`dataSource.transaction(execute, options, cb)`](https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource-prototype-transaction) offers an interface into a simple way to create transactions and bind them to models, so that the transaction object never needs to be explicitly created, passed along to Model database methods, or committed / rolled-back.
```

Comment:
Minor wording suggestion: 

"Inspired by the way [Objection.js](https://github.com/Vincit/objection.js) handles transactions, the method [`dataSource.transaction(execute, options, cb)`](https://apidocs.strongloop.com/loopback-datasource-juggler/#datasource-prototype-transaction) creates transactions and binds them to models.  Thus, you don't need to explicitly create a transaction object, pass it along to Model database methods, nor committed / rolled-back.

Reply:
Agreed, but I think I'll link to the specific page from the Objection.js link: http://vincit.github.io/objection.js/#binding-models-to-a-transaction